### PR TITLE
Fix issue #31: Canvas overlaps outside screen

### LINKED ISSUE #31: Canvas overlaps outside screen
Activity canvas overlaps to the part that's not visible on the screen; @jriyyya 

image

### PR DESCRIPTION
**Pull request by jriyyya**
@chimosky Please review the fix

Fixes #31

### CODE REVIEW

File: howtoplay.py [hunk 1]
```diff
@@ -1,27 +1,45 @@
...
-import pickle
+from gi.repository import Gdk
 from gi.repository import Gtk
+from button import Button
+
+
+def color_parse(color):
+    rgba = Gdk.RGBA()
+    rgba.parse(color)
+    return (int(rgba.red * 255), int(rgba.green * 255), int(rgba.blue * 255))
```
Comment (chimosky) [inline howtoplay.py:10]: No need to redefine this here as you already did in settings so you can import that and use.

**Comment by chimosky** [comment]
Tested, works as expected.

I'll wait for @sourabhaa to test before merging.

File: howtoplay.py (hunk 1 above)
Comment (jriyyya) [reply howtoplay.py]: done

**Comment by sourabhaa** [comment]
Thanks. Tested on multiple resolutions, works as expected.

**Comment by sourabhaa** [comment]
I noticed that the UI elements of other screens are rendered n close event. @chimosky is it related to #27 ?
Screenshot from 2023-09-03 11-06-58

**Comment by chimosky** [comment]
That's strange, didn't notice that while testing and it's not related to #27 and the text shown is for the highscore, do the logs say anything?

**Comment by jriyyya** [comment]
I also wasn't able to reproduce it

**Comment by sourabhaa** [comment]
The logs are clean, but noticed this happens when the close event is triggered from other screens as well. Since it's not related to this PR, I'll open an issue.

**Comment by chimosky** [comment]
> The logs are clean, but noticed this happens when the close event is triggered from other screens as well. Since it's not related to this PR, I'll open an issue.

What close event are you referring to?

I'll merge this as we no longer have issues with it and you'll open an issue to track what you noticed.

**Pull request merged**